lib/src/components/NxCheckbox/NxCheckbox.scss

```diff
@@ -18,3 +18,14 @@
     vertical-align: 4px;
   }
 }
+
+.nx-radio-checkbox__input:checked:disabled {
```

Comment:
```suggestion
.nx-checkbox__input:checked:disabled {
```

Reply:
This seems to have become un-fixed

Reply:
> This seems to have become un-fixed

Updated with the fix. I had to undo a commit yesterday, seems like I accidentally undid this fix as well. 
1d21fc54210e59da3b3c4d81f821950d8be89d95